src/rpc/response.rs

```diff
@@ -426,6 +426,24 @@ impl<'a> HasOwner for Result<CachedResponse, Error<'a>> {
     }
 }
 
+pub(super) fn identity_response<'a, 'b>(req_id: Id<'a>, identity: &'b str) -> HttpResponse {
+    let mut map = HashMap::new();
```

Comment:
```
#[derive(Serialize)]
struct Identity {
  identity: &str,
}
```
?

Reply:
might as well do it without HashMap, fixed :heavy_check_mark: